cmd/geth/main.go

```diff
@@ -341,6 +345,10 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 			}
 		}
 	}()
+
+	//START - QUORUM permission service
+	go startQuorumPermissionService(ctx, stack)
```

Comment:
It'd be better to capture the error and halt the node if permission service is not enabled
```suggestion
go func() {
  if permission enabled {
     if err := startQuorumPermissionService(..); err != nil {
        utils.Fatalf(...)
     }
   }
}()
```

also, even `--permissioned` flag is given, `--rpcapi` must be honored too. 
A node may only have `--permissioned` and `quorumPermission` API must not be available unless `--rpcapi quorumPermission` is provided. Has it been captured anywhere?

Reply:
done